cognite/client/_api/simulators/routines.py

```diff
@@ -217,3 +222,45 @@ def list(
             sort=[PropertySort.load(sort).dump()] if sort else None,
             filter=routines_filter.dump(),
         )
+
+    def run(
+        self,
+        routine_external_id: str,
+        inputs: Sequence[SimulationInputOverride] | None = None,
+        run_time: int | None = None,
+        queue: bool | None = None,
+        log_severity: Literal["Debug", "Information", "Warning", "Error"] | None = None,
+        wait: bool = True,
+        timeout: float = 60,
+    ) -> SimulationRun:
+        """`Run a simulation <https://developer.cognite.com/api#tag/Simulation-Runs/operation/filter_simulation_runs_simulators_runs_list_post>``
+        Args:
+            routine_external_id (str): External id of the simulator routine
+            inputs (Sequence[SimulationInputOverride] | None): List of input overrides
+            run_time (int | None): Run time in milliseconds. Reference timestamp used for data pre-processing and data sampling.
+            queue (bool | None): Queue the simulation run when connector is down.
+            log_severity (Literal['Debug', 'Information', 'Warning', 'Error'] | None): Override the minimum severity level for the simulation run logs. If not provided, the minimum severity is read from the connector logger configuration.
+            wait (bool): Wait until the simulation run is finished. Defaults to True.
+            timeout (float): Timeout in seconds for waiting for the simulation run to finish. Defaults to 60 seconds.
+        Returns:
+            SimulationRun: Created simulation run
+        Examples:
+            Create new simulation run:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> run = client.simulators.routines.run(routine_external_id="routine1", log_severity="Debug")
+        """
+        self._warning.warn()
+        run_object = SimulationRunWrite(
+            routine_external_id=routine_external_id,
+            inputs=list(inputs) if inputs is not None else None,
+            run_time=run_time,
+            queue=queue,
+            log_severity=log_severity,
+        )
+
+        simulation_run = self._cognite_client.simulators.runs.create(run_object)
+
+        if wait:
+            simulation_run.wait()
```

Comment:
did we forget to pass the `timeout` to the wait function?

Reply:
you are totally right, thanks